bun-tests/fake-snippets-api/routes/orders/order-with-circuit-json-or-package-release.test.ts

```diff
@@ -0,0 +1,69 @@
+import { getTestServer } from "bun-tests/fake-snippets-api/fixtures/get-test-server"
+import { test, expect } from "bun:test"
+
+test("create order with only circuit_json (✅)", async () => {
+  const {
+    axios,
+    seed: { order },
+  } = await getTestServer()
+
+  const response = await axios.post("/api/order_quotes/create", {
+    vendor_name: "jlcpcb",
+    circuit_json: order.circuit_json,
```

Comment:
same feedback as before, this is confusing because order quotes do not require an order

Reply:
you could just put in a small array hard-coded and that would be fine